addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/handler/SolarEdgeAggregateDataPolling.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge.handler;
+
+import org.openhab.binding.solaredge.internal.command.AggregateDataUpdate;
+import org.openhab.binding.solaredge.internal.command.AggregatePeriod;
+import org.openhab.binding.solaredge.internal.command.SolarEdgeCommand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Polling worker class. This is responsible for periodic polling of sensor data.
+ *
+ * @author Alexander Friese - initial contribution
+ */
+public class SolarEdgeAggregateDataPolling implements Runnable {
+    /**
+     * Logger
+     */
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    /**
+     * Handler for delegation to callbacks.
+     */
+    private final SolarEdgeHandler handler;
+
+    /**
+     * Constructor.
+     *
+     * @param handler handler which handles results of polling
+     */
+    public SolarEdgeAggregateDataPolling(SolarEdgeHandler handler) {
+        this.handler = handler;
+    }
+
+    /**
+     * Poll the SolarEdge Webservice one time per call.
+     */
+    @Override
+    public void run() {
+        if (handler.getWebInterface() != null) {
+            logger.debug("polling SolarEdge aggregate data {}", handler.getConfiguration());
+
+            SolarEdgeCommand adu_day = new AggregateDataUpdate(handler, AggregatePeriod.DAY);
```

Comment:
The [code guidelines](http://docs.openhab.org/developers/development/guidelines.html#a-code-style) state that we should follow the [Java naming conventions](http://java.about.com/od/javasyntax/a/nameconventions.htm)

In short:

- **Packages**: Names should be in lowercase
- **Classes/Interfaces**: Names should be in CamelCase
- **Methods/Variables**: Names should be in mixed case (CamelCase starting with lower case)
- **Constants**: Names should be in uppercase

Reply:
will fix this

Reply:
fixed